### DISCUSSION TITLE
Evden işe gitme sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#294] [Zeka Sorusu]
Mehmet Bey İstanbul’da çalışır, adalarda oturur. Her gün işten çıkınca vapura biner adaya gider. Vapur limana vardığında Mehmet beyin şöförü de aynı anda limanda olur. Mehmet bey vapurdan iner arabasına biner ve evine gider.

Mehmet bey bir gün işten 1 saat erken çıkar ve herzaman bindiği vapurdan 1 saat önce kalkan vapura biner. Adada limana vardığında şöförü yoktur. Çünkü erken çıktığını haber vermemiştir. Eve doğru yürümeye karar verir. Yolda şöförüyle karşılaşır. Arabaya biner ve evine gelir. Saate baktığında her zamankinden 20 dakika önce evde olduğunu görür. Soru şu. Mehmet Bey kaç dakika yürümüştür?

**Comment by xBURAKx2011** [comment]
Mehmet Bey 40 dakika yürümüştür.

**Comment by yigidi** [comment]
Mehmet Bey 40 dk yürümüştür. Çünkü vapurdan indiğinde her zamankinden 60 dk önce oradadır. 20 dk kala evde olduğuna göre 60-20 =40
